pswalker/plans.py

```diff
@@ -255,6 +256,9 @@ def measure(detectors, num=1, delay=None, filters=None, drop_missing=True):
     drop_missing : bool, optional
         Choice to include events where event keys are missing
 
+    max_drops : int, optional
```

Comment:
The docstring added here has the wrong keyword: max_drops instead of max_dropped